src/anshuKJun20/PalindromeNumber.java

```diff
@@ -0,0 +1,28 @@
+package anshuKJun20;
+
+public class PalindromeNumber {
+	private void findPalindromeNumber(int number) {
+		int number1 = number, add = 0;
+		while (number > 0) {
+			int remainder = number % 10;
+			number = number / 10;
+			add = add * 10 + remainder;
```

Comment:
You can optimize the code by merging line 7 in line 9.
i.e. At line 9 write add = (add *10)+(number%10);
Remove line 7

Reply:
Changes done. Thankyou for reviewing it.